impress-org/givewp
# Feature: Scroll Stripe checkout modal into view for Legacy + Multistep forms

Resolves #6788

## Description

This PR adds functionality to scroll the Stripe checkout modal into view when the user clicks the purchase button. The checkout modal will now be visible to the user, regardless of their screen size. This fix currently only works for legacy & classic templates.

## Affects

Stripe checkout modal

## Visuals



## Testing Instructions

- Create a form and set Stripe Checkout as a payment option.
- Begin the donation process on a mobile device, a long donation form, or with altered viewports on a desktop.
- Fill out the form until you reach the second to last step before payment.
- Click on the "Donate Now" button.
- The Stripe modal should scroll into view.






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204508757292153
